.eleventy.js

```diff
@@ -5,20 +5,21 @@ const fs = require("fs");
 // Import filters
 const dateFilter = require('./src/filters/date-filter.js');
 const markdownFilter = require('./src/filters/markdown-filter.js');
+const permalinkFilter = require('./src/filters/permalink-filter.js');
 const w3DateFilter = require('./src/filters/w3-date-filter.js');
 
 // Import transforms
 const htmlMinTransform = require('./src/transforms/html-min-transform.js');
 const parseTransform = require('./src/transforms/parse-transform.js');
 
-
 // Import data files
 const site = require('./src/_data/site.json');
 
 module.exports = function(config) {
   // Filters
   config.addFilter('dateFilter', dateFilter);
   config.addFilter('markdownFilter', markdownFilter);
+  config.addFilter('post_permalink', permalinkFilter);
```

Comment:
Add our new, custom filter, name it "post_permalink" (feel free to rename it if you like "wp_permalink" or something better), and then set the function to our new `permalinkFilter` variable above from L8.